samples/basic/blink_led/boards/nrf52810_pca10040.conf

```diff
@@ -0,0 +1,3 @@
+# The nrfx Hardware PWM driver cannot be used in this sample since the driver
+# does not allow to set a period longer than ~262 ms.
+CONFIG_PWM_NRFX=n
```

Comment:
Can you add to this comment that you're disabling the HW PWM driver in order to enable the SW PWM driver?

Reply:
Done.